trackhubs/management/commands/enrich.py

```diff
@@ -77,15 +69,13 @@ def handle(self, *args, **options):
         # Get the trackdb ID if provided
         trackdb_id = options['trackdbid'] if options['trackdbid'] is not None else None
         if options['trackdbid'].lower() == 'all':
```

Comment:
Use trackdb_id instead?

Reply:
It will prompt this error:
```
$ python manage.py enrich    
usage: manage.py enrich [-h] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH] [--traceback] [--no-color] [--force-color] [--skip-checks] trackdb_id
manage.py enrich: error: the following arguments are required: trackdb_id
```

Reply:
In that case,

Reply:
You're right @sanjay-boddu, I got rid of it